pkg/minikube/reason/reason.go

```diff
@@ -207,7 +207,13 @@ var (
 	ProviderNotFound    = Kind{ID: "PROVIDER_NOT_FOUND", ExitCode: ExProviderNotFound}
 	ProviderUnavailable = Kind{ID: "PROVIDER_UNAVAILABLE", ExitCode: ExProviderNotFound, Style: style.Shrug}
 
-	DrvCPEndpoint         = Kind{ID: "DRV_CP_ENDPOINT", ExitCode: ExDriverError}
+	DrvCPEndpoint = Kind{ID: "DRV_CP_ENDPOINT",
+		Advice: `Recreate the cluster by running:
+		minikube delete{{.profile}}
```

Comment:
how would the output look like if the profile is not minikube?
should it include minikube delete -p {{.profile}}  ?

Reply:
good catch! fixed it, and updated output in the PR description